### PR TITLE
[Bexley] Add communal/no service container request form.

### PR DESCRIPTION
**Pull request by dracos**
[skip changelog]
Fixes https://github.com/mysociety/societyworks/issues/4585

### CODE REVIEW

**Review by nephila-nacrea — CHANGES_REQUESTED** [review]
I suspect there's a check somewhere where we redirect back to bin days if there's no services for an enquiry etc. In any case, for a no-service property, clicking the 'Order bins or boxes' button takes me right back to bin days.

File: templates/web/bexley/waste/_more_services_sidebar.html [hunk 1]
```diff
@@ -8,7 +8,13 @@ <h3>More services</h3>
     <li><a href="[% c.uri_for_action('waste/garden', [ property.id ]) %]">Subscribe to [% c.cobrand.garden_service_name %]</a></li>
   [% END %]
   <li><a [% external_new_tab | safe %] href="https://mybexley.bexley.gov.uk/service/rubbish_and_recycling_enquiries">Report an issue with your collection service</a></li>
-  <li><a [% external_new_tab | safe %] href="https://www.bexley.gov.uk/services/rubbish-and-recycling/find-your-collection-day-and-report-missed-bin/replace-damaged-or-missing-bins">Order replacement bins, boxes or sacks</a></li>
+
+  [% IF property.is_communal %]
+    <li><a href="[% c.uri_for_action('waste/enquiry', [ property.id ]) %]?category=Request+container+enquiry">Order replacement bins, boxes or sacks</a></li>
+  [% ELSE %]
```
[the remark this answers is not in the document]
Comment (dracos) [reply templates/web/bexley/waste/_more_services_sidebar.html:14]: It appears in the `templates/web/bexley/waste/_bin_days_no_collections.html` unless I've misunderstood.

**Comment by dracos** [comment]
Hmm, my test does check 10006 which is a no-services property and is displaying the 'no collections' page with a link that it is following to then request the form. Do you have the "Request container enquiry" category existing? The enquiry flow will redirect to the bin day page if the requested category does not exist.

File: templates/web/bexley/waste/_more_services_sidebar.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (nephila-nacrea) [reply templates/web/bexley/waste/_more_services_sidebar.html:14]: I think before your rebase it was showing me an 'Order replacement' link in the 'More services' sidebar for no-services, but that looks fine now.

**Review by nephila-nacrea — APPROVED** [review]

**Pull request merged**